roles/pulp/templates/pulp.container.j2

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Pulp 
+After=network.target
+
+[Container]
+Image={{ pulp_image }}
+ContainerName={{ pulp_container_name }}
+PublishPort={{ pulp_ports }}
```

Comment:
This also throws errors... I think to start with we need to go the route I did with Candlepin, and use `Network=host`.

Reply:
Ignore that comment, `Network=host` breaks the all-in-one Pulp container because it has a postgres within the container and it fails looking for the postgresql running on the host.